### ISSUE TITLE
after usage of `repeat.for` attribute, invalid binding interpolations are not reported

### ISSUE THREAD

**Issue by atsu85** [MeirionHughes/aurelia-template-lint#73]
example:
```html
<template>
      \${existing} OK
      \${missing1} ERROR reported as expected
      <a repeat.for="item of items">
          \${missing2} ERROR NOT REPORTED
          \${item}
      </a>
      \${missing3} ERROR NOT REPORTED
</template>
```

I'll send a PR with failing test soon

**Comment by atsu85** [comment]
created [PR](https://github.com/MeirionHughes/aurelia-template-lint/pull/75) with failing test to demonstrate this issue

**Comment by MeirionHughes** [comment]
this looks like its because its implicitly typed; i.e. `items = []` fails, ` items:Boo[] = [];` works
